frontend/src/views/ElementView.vue

```diff
@@ -44,9 +44,20 @@
         <div class="fr-text--sm !mb-0 capitalize">{{ einecNumber }}</div>
       </div>
 
-      <div v-if="usefulParts && usefulParts.length" class="col-span-12 sm:col-span-4 md:col-span-3 flex flex-col mt-4">
+      <div v-if="plantParts && plantParts.length" class="col-span-12 sm:col-span-4 md:col-span-3 flex flex-col mt-4">
         <div class="fr-text--sm !font-medium !mb-1">Parties utiles</div>
-        <DsfrTag small :label="part" v-for="part in usefulParts" :key="part" class="mb-1 capitalize"></DsfrTag>
+        <DsfrTag small :label="part" v-for="part in plantParts" :key="part" class="mb-1 capitalize"></DsfrTag>
+      </div>
+
+      <div v-if="plantParts && plantParts.length" class="col-span-12 sm:col-span-4 md:col-span-3 flex flex-col mt-4">
```

Comment:
```suggestion
      <div v-if="toWatchParts && toWatchParts.length" class="col-span-12 sm:col-span-4 md:col-span-3 flex flex-col mt-4">
```
Peut-être que je ne montrerais pas cette section si on n'a pas de parties à surveiller pour ne pas avoir une colonne vide, t'en penses quoi ? 

![image](https://github.com/betagouv/complements-alimentaires/assets/1225929/ea274bb4-8064-43e2-a59d-04d08070cd9c)